adapter_pipelines/Optimus/adapter_example_bundle_specific_v3.json

```diff
@@ -1,4 +0,0 @@
-{
-  "AdapterOptimus.bundle_uuid": "c1677888-59fb-43bd-b3e3-05c274cdb11a",
-  "AdapterOptimus.bundle_version": "2017-09-21T173007.598036Z"
-}
```

Comment:
If we removed these v3 files for Optimus, I think it would be great we removed other v3 files for 10x and ss2, for a better consistency. What do you think?

Reply:
That makes sense to me!